Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -548,7 +562,13 @@ function Add-DatabaseLogin([string] $userName, [string] $connString)
                 set @sql = 'CREATE LOGIN ' + QUOTENAME('$userName') + ' FROM WINDOWS'
                 EXEC sp_executesql @sql
             END"
-    Invoke-Sql $connString $query @{userName=$userName} | Out-Null
+    try {
+        Invoke-Sql $connString $query @{userName=$userName} | Out-Null
+    }
+    catch {
+        Write-Error $_.Exception -ErrorAction Continue
```

Comment:
Is there a reason we are stopping for some and continuing for other?

Reply:
The original `Install-Identity-Windows` script had try/catch blocks for some `Invoke-Sql` calls while omitted it on others. Do we always want to halt if we can't do any SQL-related activity, including, for example, migrating data?